apps/functional_chat/app/components/chat/ChatInferenceModule.tsx

```diff
@@ -14,6 +14,7 @@ import { ChatScrollAnchor } from './ChatScrollAnchor';
 import Toggle from './toggle';
 import Markdown from './markdown';
 import { EmptyLLMOutput } from './empty-llm-state';
+import axios from 'axios';
```

Comment:
Lets just use fetch() for consistency so we don't mix multiple ways of calling the backend in the same codebase.